CHANGELOG.md

```diff
@@ -4,6 +4,12 @@ All notable changes to this project are documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [v1.2.0] 2020-02-14
```

Comment:
Should dropping support for Python 2 not be a "major" semver release, 2.0.0? Or are we following "Cumulus" semver conventions here?

Reply:
You are correct, however I believe all efforts on this project are not following semver.

Reply:
Is there any reason that this repo could not? I guess I'd rather follow actual semver in the places where we can.

Reply:
Agreed, however project terms/rules dictate sadly.